src/pages/Applications/components/Application.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { IconType } from 'react-icons';
+
+import { useWorkspace } from '@/hooks/useWorkspace';
+
+interface ApplicationProps {
+  name: string;
+  description: string;
+  icon: IconType;
+}
+
+const Application: React.FC<ApplicationProps> = ({ name, description, icon: Icon }) => {
+  const { setActiveApplication } = useWorkspace();
+
+  return (
+    <button className="application-card" onClick={() => setActiveApplication(name)}>
+      <div className="icon">
+        <Icon />
+      </div>
+      <div className="app-details">
+        <h3>{name}</h3>
+        <p>{description}</p>
+      </div>
+    </button>
+  );
+};
+
+export default Application;
```

Comment:
Same here on the `named` vs `default` exports, also think `React.FC` is generally falling by the wayside these days and that components are now being written as:

```JavaScript
export const Application = ({ name, description, icon: Icon }: ApplicationProps) ={...}
```
Here is a reddit article on this https://www.reddit.com/r/reactjs/comments/wjq51d/is_reactfc_not_recommended_what_are_other/, I also googled `should I use React.FC` and got a number of pros/cons, but one that caught my eye was PR in create-react-app, by facebook themselves stating `React-FC` is unecessary https://github.com/facebook/create-react-app/pull/8177. I don't think it is a big deal to stop this PR getting merged, but I suggest you no longer use `React.FC` for reasons given in links above.

Reply:
Thanks for the link, I've updated this